Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -5969,6 +5969,31 @@ public void ActBrowserElementHandler(ActBrowserElement act)
                     break;
                 case ActBrowserElement.eControlAction.CloseAll:
                     Driver.Quit();
+                    break;
+                case ActBrowserElement.eControlAction.GetBrowserLog:
+
```

Comment:
Please add unit test to cover this.
Also send me the screenshot of how it looks in the output values

Reply:
added unit test